portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,17 +42,20 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  // private ArrayList<String> comments=new ArrayList<String>();
-
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     final String comment = request.getParameter("comment");
-    // comments.add(comment);
     Entity commentEntity = new Entity("comment");
-    long timestamp = System.currentTimeMillis();
+    Document doc = Document.newBuilder().setContent(comment).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    final long timestamp = System.currentTimeMillis();
 
     commentEntity.setProperty("comment", comment);
     commentEntity.setProperty("timestamp", timestamp);
+    commentEntity.setProperty("score", score);
+    System.out.println(score);
```

Comment:
Are you logging this for debugging? Prefer java.util.logging.Logger.

And you'd better add some description for the score, e.g "The comment's sentiment score is " + (String) score.